core/test/unit/config_spec.js

```diff
@@ -871,8 +869,7 @@ describe('Config', function () {
             // Run the test code
             config.checkDeprecated();
 
-            logStub.calledOnce.should.be.true;
-            logStub.calledWithMatch('mail.fromaddress').should.be.true;
+            logStub.calledOnce.should.be.false;
```

Comment:
As in, the config.set doesn't work here because _.merge does not take keys with `undefined` values into account.